src/methods.jl

```diff
@@ -207,6 +208,16 @@ julia> trim(frame, "[2:5, 1:5]")
 trim(frame::AbstractArray, idxs) = copy(trimview(frame, idxs))
 
 
+"""
+    trim(::FITSIO.ImageHDU, idxs)
+    trim(filename, idxs; hdu=1)
+
+Load a FITS file or HDU before trimming.
+"""
+trim(frame::ImageHDU, idxs) = trimview(getdata(frame), idxs)
+trim(frame::String, idxs; hdu = 1) = trim(FITS(frame)[hdu], idxs)
```

Comment:
please add a method (and relevant documentation) for getting the idxs as a header key. This should be exactly the same as `subtract_overscan`.